app/utils/chartTypeUtils.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Chart types registry
+ */
+import update from 'react-addons-update';
+import { selectableChartTypes } from '../constants/chartTypes';
+
+const nvd3Defaults = {
+  nvd3SingleSeries: {
+    x: (point) => point.label,
+    y: (point) => point.value,
+    height: 400,
+  },
+  nvd3MultiSeries: {
+    x: (point) => point.x,
+    y: (point) => point.y,
+    height: 400,
+  },
+};
+
+export function getChartTypeObject(type) {
+  for (const typeObj of selectableChartTypes) {
```

Comment:
`for ... of` loops are not supported in < IE 12 so this should probably be switched to something else.

```
return Object.keys(selectableChartTypes).reduce((prev, key) =>
  type === selectableChartTypes[key].config.type ? 
    selectableChartTypes[key]  :
    prev,
  null);